mattermost-plugin/server/manifest.go

```diff
@@ -13,8 +13,8 @@ var manifest *model.Manifest
 const manifestStr = `
 {
   "id": "focalboard",
-  "name": "Focalboard",
-  "description": "This provides focalboard integration with mattermost.",
+  "name": "Mattermost Boards",
```

Comment:
These fields should have been updated in another PR, just adding those as they're autogenerated on plugin build.